### DISCUSSION TITLE
misgdbr for yeast

### DISCUSSION THREAD

**Discussion by kiangaikani** [igordot/msigdbr#47] [Q&A] [answered]
Hi, I am using the package for yeast GSEA, and I see some enrichments that seem not to be related to yeast, such as:
HP_ADDICTIVE_BEHAVIOR or HP_ACUTE_MYELOID_LEUKEMIA. I am a beginner; could you please tell me if the error is from my end I am doing something wrong?

I used

```
#get all collections/signatures with yeast
yeast_gsea <- msigdbr(species = "Saccharomyces cerevisiae")
yeast_gsea %>%   dplyr::distinct(gs_cat, gs_subcat) %>%   dplyr::arrange(gs_cat, gs_subcat)
#choose a specific msigdb collection/subcollection
yeast_gsea_c5 <- msigdbr(species = "Saccharomyces cerevisiae", category = "C5") %>% dplyr::select(gs_name, gene_symbol)

```

**Comment by igordot** [comment, marked as answer]
Most MSigDB gene sets are based on human or mouse experiments. All the species conversions in msigdbr are done on a gene level. The specific gene sets may or may not be relevant to every species. To use your example, addictive behavior or leukemia may not be present in yeast, but it's possible that your particular experiment has a signature that is similar to those mammalian processes.

**Comment by kiangaikani** [comment]
Thanks for the clarification.
